src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/xlf/MicrosoftNetCoreAnalyzersResources.cs.xlf

```diff
@@ -552,6 +567,21 @@
         <target state="translated">{0} zakazuje TLS 1.2 a povoluje SSLv3.</target>
         <note />
       </trans-unit>
+      <trans-unit id="DoNotDisableTokenValidationChecksDescription">
+        <source>Token validation checks ensure that while validating tokens, all aspects are analyzed and verified. Turning off validation can lead to security holes by allowing untrusted tokens to make it through validation.</source>
+        <target state="new">Token validation checks ensure that while validating tokens, all aspects are analyzed and verified. Turning off validation can lead to security holes by allowing untrusted tokens to make it through validation.</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="DoNotDisableTokenValidationChecksMessage">
+        <source>TokenValidationParameters.{0} should not be set to false as it disables important validation.</source>
```

Comment:
@Evangelink This shouldn't end in a period right?

Reply:
All the message elements I saw ended with a period. The titles had none and I tried to match that style there too. If there's a different consensus on style I can always push an update.

Reply:
@TimHannMSFT, I believe the message should only end in a period when multi-statement, and doesn't if single-statement.

There is already an analyzer for that, but it either got confused by the `.{0}` and considered it a multi-statement or it's not working for some other reason.

The rule for that is:

https://github.com/dotnet/roslyn-analyzers/blob/aac24788f059850f12978bc50b80444607e1f21c/src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx#L505-L507

Reply:
The analyzer is outdated in 2.9.x branch.